src/common/config/store/local.go

```diff
@@ -0,0 +1,49 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package store
+
+import (
+	"sync"
+
+	"github.com/goharbor/harbor/src/common/config"
+)
+
+func init() {
+	config.RegistStore("local", &Local{})
+	config.RegistStore("remote", &Remote{})
+}
+
+// Local - Local config store
+type Local struct {
```

Comment:
I recall the LocalStore should reference an instance of `Driver` for accessing local raw data?